webapi/src/main/scala/org/knora/webapi/responders/admin/GroupsResponderADM.scala

```diff
@@ -241,6 +237,7 @@ final case class GroupsResponderADMLive(
    * @return A sequence of [[User]]
    */
   private def groupMembersGetADM(
+    // TODO: change to GroupIri, refactor name and simplify
```

Comment:
**todo**: Please do this simple change immediately or remove the `TODO` comment

IMHO such comments are helpful when working in your local branch. But we do not have a process in place to take care of such comment. Hence, these comments tend to cause noise. If you want to do it in the next PR feel free to remember this in your personal notes.

Reply:
Done in [12de345](https://github.com/dasch-swiss/dsp-api/pull/3075/commits/12de3453bea0c4b09ff9217ad4cfff4772ca9f3c)